ops/model.py

```diff
@@ -4435,6 +4436,7 @@ class CloudSpec:
     is_controller_cloud: bool = False
     """If this is the cloud used by the controller, defaults to False."""
 
+    # Note that this is public because it was public in model.py.
```

Comment:
Tony says that those comments should not be there

Reply:
```suggestion
```